service-discovery/README.adoc

```diff
@@ -31,304 +31,399 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
 
-OpenShift is an Enteprise Kuberentes Platform. When deploying Quarkus and any other type of application, developers can use
-the https://developers.redhat.com/blog/2019/12/19/introducing-the-service-binding-operator/[Service Binding Operator] to
-enable your applications to discover service configuration automatically in your OpenShift or Kubernetes instance.
+For example, suppose that you have the following applications deployed on OpenShift:
 
-You can review all possible commands in https://github.com/redhat-developer/app-services-cli/blob/main/docs/commands/rhoas_cluster.adoc[cli documentation].
+* One application that publishes price updates for a variety of stocks
+* A second application that consumes the price updates for publication on a web page
 
+In addition, you have a Kafka instance in {product}. Each time the first application produces a price update, you want to use the Kafka instance to forward the update as an event to the second, consuming application.
 
-In this Quick Start, you'll connect your OpenShift Streams for Apache Kafka cloud service to your OpenShift cluster using the Red Hat OpenShift Application Services. (RHOAS) Operator and CLI. For more information on Red Hat OpenShift Application Services, please visit link:https://cloud.redhat.com/application-services[cloud.redhat.com.]
+To achieve this behavior, you need a way to connect the applications in OpenShift to your Kafka instance in {product}. Connecting a Kubernetes application to a backing service such as {product} is called __service binding__.
 
-.Prerequisites
-ifndef::community[]
-* You have a Red Hat account.
-endif::[]
-* You have a running Kafka instance in {product}.
+This guide describes how to use the Red Hat OpenShift Application Services (RHOAS) Operator to perform service binding for {product}. The Kubernetes platform used throughout this guide is Red Hat OpenShift.
 
-// Condition out QS-only content so that it doesn't appear in docs.
-// All QS anchor IDs must be in this alternate anchor ID format `[#anchor-id]` because the ascii splitter relies on the other format `[id="anchor-id"]` to generate module files.
-ifdef::qs[]
-[#description]
-Using the {product-long} services in your OpenShift cluster{product-long}.
 
-[#introduction]
-Welcome to the {product-long} Service Discovery quick start. In this quick start, you'll learn how to connect your {product} services with https://quarkus.io/[Quarkus^]. Tutorial will connect Kafka with your Application to produce and consume messages from your Kafka instances in {product}.
-endif::[]
+[id="con-about-service-binding-using-rhoas-operator_{context}"]
+== About service binding using the RHOAS Operator
+
+Connecting a Kubernetes application to a backing service such as {product-long} is called __service binding__. To perform service binding for {product}, you use the Red Hat OpenShift Application Services (RHOAS) Operator.
```

Comment:
My only slight issue here is that the general concept of connecting an application deployed to Kubernetes is not called service binding. What is called service binding is the way we recommend you do it in this guide.

Also I think it should an "application running on Kubernetes" or similar. A "Kubernetes application" suggests something that is part of Kubernetes.

Reply:
@pmuir Happy with this rephrasing?

"In App Services, using Operator functionality to automatically create a connection between a Kafka instance and an application running on Kubernetes is called _service binding_."

Reply:
That's much better, nice one @jbyrne-redhat